src/components/dashboard/Dashboard.js

```diff
@@ -426,7 +403,7 @@ const Dashboard = props => {
 
     // calculate left margin for aligning the balance to the right
     // - 20 is to give more space to the number, otherwise (in native) it gets cut on the right side
-    const balanceCalculatedLeftMargin = headerContentWidth - balanceBlockWidth - 20
+    const balanceCalculatedLeftMargin = headerContentWidth - balanceBlockWidth.current - 20
```

Comment:
why changed to ref ? effect won't work on balanceBlockWidth now

Reply:
we don't need to effect on balanceBlockWidth, we just need its value